src/app/js/root-admin/Tenants.js

```diff
@@ -378,12 +409,13 @@ const Tenants = ({ handleLogout }) => {
             flex: 1,
             renderCell: (params) => {
                 return (
-                    <Button
+                    <IconButton
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏What's the difference between `Button` and `IconButton` ?

Reply:
You can learn for here:
https://mui.com/material-ui/react-button/#icon-button